internal/pkg/cli/run_local.go

```diff
@@ -559,6 +595,47 @@ func (o *runLocalOpts) getSecret(ctx context.Context, valueFrom string) (string,
 	return getter.GetSecretValue(ctx, valueFrom)
 }
 
+type host struct {
+	host string
+	port string
+}
+
+type hostDiscoverer struct {
+	ecs  ecsLocalClient
```

Comment:
I know you didn't name this! What do you think of renaming `ecsLocalClient` to maybe, `ecsClient`? I don't know what is "local" about it 😂